### ISSUE TITLE
[assistance] How to actually import library and how to make new users?

### ISSUE THREAD

**Issue by Z3ROGRAV1TYx** [happypandax/happypandax#132]
I cannot for the life of me figure out how to import my folders for it to actually grab all my files. I see no option to browse for a folder or anything as far as actually setting up the files go.

I also cannot figure out how to create new users (Because I disabled default user and guest access). I notice in the docs it says to use the command line arguments, but I can't figure out how to open up a command prompt for HPX? Is this if I was using GNU/Linux? Currently running it on Windows 10 (Used the Installer not zip).

**Comment by GentleViking** [comment]
Click the green plus sign on the left sidebar of the webclient. After that, at the top there should be a tab labeled scan. This is where you put in the directory path you wish to scan. 

I also need help figuring out how to add users.

Sorry for the vague instructions, I'm currently on my phone. I'll give you more detailed instructions if no one else responds.

**Comment by Z3ROGRAV1TYx** [comment]
@GentleViking No worries, I appreciate it. I won't be able to confirm the first part until I get a user account profile setup so I'll be waiting on you or someone else to explain that :)
Thank you!

**Comment by Z3ROGRAV1TYx** [comment]
Still having trouble figuring out how to create users >.<

**Comment by twiddli** [comment]
Sorry for the late response.

I'll make it easier to create users,

For now:
1. Open a command prompt
2. Find the path where you installed HPX (if installed with the default installation path, this will be `C:\Users\{username}\AppData\Local\HappyPanda X`, replace `{username}` with your pc account name)
3. In the command prompt enter this `"path/to/hpx" --create-user` and then follow the instructions this command prints out (replace `path/to/hpx` with the actual path to HPX, make sure it remains in qoutes)

**Comment by Z3ROGRAV1TYx** [comment]
@twiddli I seem to have gotten my user info setup and all working thanks to your instructions thank you!!

@GentleViking Upon using your instructions to scan my galleries, all seems to be working well!

I do have a few questions/issues that you or Twiddli might be able to answer, it's mostly regarding metadata, I'm going to open a new issue for this because it's a few things. Hope that's no problem! Thank you!
